### PR TITLE
8283279: [Testbug] Improve TestGetSwapSpaceSize

### PR DESCRIPTION
**Pull request by jerboaa**
Please review this container test improvement. The test in question only makes sense iff the total swap space size as reported by the container aware OperatingSystemMXBean is `0`. If that's not the case, then something else might be amiss, e.g. OperatingSystemMXBean reporting the host swap limits. In such a case a passing test tells us nothing. Certainly not if the
fix from [JDK-8242480](https://bugs.openjdk.java.net/browse/JDK-8242480) is present or not.

Testing:
- [x] Manual with and without the code fix of JDK-8242480. Still passes with the fix, and fails without. Tested the test on cgroups v1 and cgroups v2.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8283279](https://bugs.openjdk.java.net/browse/JDK-8283279): [Testbug] Improve TestGetSwapSpaceSize

### Reviewers
 * [Jie Fu](https://openjdk.java.net/census#jiefu) (@DamonFool - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/7854/head:pull/7854` \
`$ git checkout pull/7854`

Update a local copy of the PR: \
`$ git checkout pull/7854` \
`$ git pull https://git.openjdk.java.net/jdk pull/7854/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 7854`

View PR using the GUI difftool: \
`$ git pr show -t 7854`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/7854.diff">https://git.openjdk.java.net/jdk/pull/7854.diff</a>

</details>

### CODE REVIEW

**Comment by jerboaa** [comment]
@DamonFool Do you think you can have a look at this? Thanks!

File: test/jdk/jdk/internal/platform/docker/GetFreeSwapSpaceSize.java [hunk 1]
```diff
@@ -24,16 +24,36 @@
 import com.sun.management.OperatingSystemMXBean;
 import java.lang.management.ManagementFactory;
 
+// Usage:
+//   GetFreeSwapSpaceSize <memoryAlloc> <memory> <memorySwapAlloc> <expectedSwap>
```
Comment (DamonFool) [inline test/jdk/jdk/internal/platform/docker/GetFreeSwapSpaceSize.java:28]: I would suggest
```
//   GetFreeSwapSpaceSize <memoryAlloc> <expectedMemory> <memorySwapAlloc> <expectedSwap>
```

File: test/jdk/jdk/internal/platform/docker/GetFreeSwapSpaceSize.java [hunk 2]
```diff
@@ -24,16 +24,36 @@
...
 public class GetFreeSwapSpaceSize {
     public static void main(String[] args) {
-        System.out.println("TestGetFreeSwapSpaceSize");
+        if (args.length != 4) {
+            throw new RuntimeException("Unexpected arguments. Expected 2, got " + args.length);
```
Comment (DamonFool) [inline test/jdk/jdk/internal/platform/docker/GetFreeSwapSpaceSize.java:32]: Shouldn't be `Expected 4` ?

**Review by DamonFool — COMMENTED** [review]
Please also update the copyright year.
Thanks.

File: test/jdk/jdk/internal/platform/docker/GetFreeSwapSpaceSize.java (hunk 1 above)
Comment (jerboaa) [reply test/jdk/jdk/internal/platform/docker/GetFreeSwapSpaceSize.java]: Thanks, fixed.

File: test/jdk/jdk/internal/platform/docker/GetFreeSwapSpaceSize.java (hunk 2 above)
Comment (jerboaa) [reply test/jdk/jdk/internal/platform/docker/GetFreeSwapSpaceSize.java]: Yes. Fixed now.

**Comment by jerboaa** [comment]
Thanks for the review, @DamonFool! Updated the patch.

**Review by DamonFool — APPROVED** [review]
LGTM
Thanks for the update.

**Comment by jerboaa** [comment]
Thanks for the review!

**Pull request closed without merging**